spec/controllers/api/v1/parks_controller_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::ParksController, type: :controller do
+  let!(:first_park) { Park.create(
```

Comment:
I recommend you use `create!` to make your tests raise an Exception so they're noisy